css-three-column-layout/styles.css

```diff
@@ -0,0 +1,65 @@
+body {
+  background-color: rgb(61 158 255);
+  color: white;
+}
+
+.container {
+  width: 1000px;
+  text-align: center;
+  margin-left: auto;
+  margin-right: auto;
+  line-height: 150%;
+  font-family: Lato, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  src:
+    url("https://fonts.gstatic.com/s/lato/v15/S6uyw4BMUTPHjx4wXiWtFCc.woff2")
+    format("woff2");
+  unicode-range:
+    u+0000-00ff,
+    u+0131,
+    u+0152-0153,
+    u+02bb-02bc,
+    u+02c6,
+    u+02da,
+    u+02dc,
+    u+2000-206f,
+    u+2074,
+    u+20ac,
+    u+2122,
+    u+2191,
+    u+2193,
+    u+2212,
+    u+2215,
+    u+feff,
+    u+fffd;
+}
+
+.row {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  margin-top: 30px;
+  margin-bottom: 30px;
+}
+
+.column-third {
```

Comment:
If the width of the column will be `25%` I would use that in the name, something like `column-fourth`